rest/model/task_annotations.go

```diff
@@ -190,3 +196,14 @@ func GetJiraTicketFromURL(URL string) (*thirdparty.JiraTicket, error) {
 
 	return nil, nil
 }
+
+func ValidateIssues(catcher grip.Catcher, issues []APIIssueLink) grip.Catcher {
+	for _, issue := range issues {
+		catcher.Add(util.CheckURL(utility.FromStringPtr(issue.URL)))
+		score := utility.FromFloat32Ptr(issue.ConfidenceScore)
+		if score <= 0 || score >= 100 {
+			catcher.Errorf("invalid confidence score '%f'", score)
```

Comment:
This should give a bit more information (that the score must be between 0 and 100); also technically 0 and 100 are valid (although wild) scores

Reply:
i said it should be in between but still allowed 0 and 100